.mergify.yml

```diff
@@ -183,6 +183,24 @@ pull_request_rules:
           2. Open a follow-up PR to update the JavaScript client `@solana/web3.js` ([example](https://github.com/solana-labs/solana-web3.js/pull/2868/files))
 
           Thank you for keeping the RPC clients in sync with the server API @{{author}}.
+  - name: Reminder to add Firedancer team to changes in `programs/`
+    conditions:
+      - or:
+        - files~=^programs/address-lookup-table/src/.*\.rs$
+        - files~=^programs/bpf_loader/src/.*\.rs$
+        - files~=^programs/compute_budget/src/.*\.rs$
+        - files~=^programs/config/src/.*\.rs$
+        - files~=^programs/loader-v4/src/.*\.rs$
+        - files~=^programs/stake/src/.*\.rs$
```

Comment:
What about the stake and vote files in `sdk/program`?

Reply:
That's a great question. I wasn't sure if Firedancer was sensitive to those changes -- @topointon-jump, what do you think?

Reply:
These seem to be mostly data structures, helper methods and tests. We are sensitive to the first two for data structures used in the native programs.

Reply:
Added these to [ce8451f](https://github.com/anza-xyz/agave/pull/2233/commits/ce8451f2f24e2c0debe9bb3c43c34ed5310d9d96)